### ISSUE TITLE
For images that do have have a title, use the alt text

### ISSUE THREAD

**Issue by TerryHowe** [TerryHowe/JeepTech#358]
For images that do not have a title, use the alt text. For example: `![D18 front](../../img/xfer/d18f.jpg)` should be changed to `![D18 front](../../img/xfer/d18f.jpg "D18 front")`
